### PR TITLE
Fix/remove not needed fetch mock

### PR DESCRIPTION
**Pull request by Moro-Code**
# Summary | Résumé

No issue number for this. Removing unused fetch mock from the template API handler tests in order to not cause confusion for anyone writing tests for handlers in the future.

# Test instructions | Instructions pour tester la modification

Changed the tests so run those :)

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

N/A

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [ ] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [x] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [x] Have you removed unnecessary lines such as `console.log` statements?
- [x] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [ ] Have you modified the change log and updated any relevant documentation?
- [x] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [ ] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: migrations/migrations/005-templates-bearer-token.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const alterTemplatesTableToAddBearerToken = `
```
Comment (jeberhardt) [inline migrations/migrations/005-templates-bearer-token.js:1]: This file seems unrelated to this PR.

Comment (Moro-Code) [reply migrations/migrations/005-templates-bearer-token.js:1]: did not update the branch before creating it from main. Awesome catch !
